src/OpenLoco/Windows/ObjectSelectionWindow.cpp

```diff
@@ -88,7 +88,7 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         makeWidget({ 1, 1 }, { 598, 13 }, WidgetType::caption_25, WindowColour::primary, StringIds::title_object_selection),
         makeWidget({ 585, 2 }, { 13, 13 }, WidgetType::buttonWithImage, WindowColour::primary, ImageIds::close_button, StringIds::tooltip_close_window),
         makeWidget({ 0, 65 }, { 600, 333 }, WidgetType::panel, WindowColour::secondary),
-        makeWidget({ 3, 15 }, { 589, 50 }, WidgetType::wt_5, WindowColour::secondary),
+        makeWidget({ 3, 15 }, { 589, 50 }, WidgetType::slider, WindowColour::secondary),
```

Comment:
I think this should now be wt_6?

Reply:
You're right! It is just used as a frame/canvas in this window.

Reply:
```suggestion
        makeWidget({ 3, 15 }, { 589, 50 }, WidgetType::wt_6, WindowColour::secondary),
```